lib/rewrite_rate_limit_headers.rb

```diff
@@ -21,6 +21,10 @@ def call(request_env)
         # (We add 5 extra seconds for safety, in case the client & server clocks are out of sync)
         headers["RateLimit-Reset"] = Time.at(headers["X-RateLimit-Reset"].to_i + 5).rfc2822
       end
+      if headers["RateLimit-Remaining"] == "0"
+        limit_expires_at = Time.rfc2822(headers["RateLimit-Reset"]).localtime
+        print "[rate limited until #{limit_expires_at.strftime '%H:%M'}] "
```

Comment:
Suggest we make it clear that the script will automatically pick up again when the rate limit expires, e.g.

```suggestion
        print "GitHub rate limit reached. bulk-changer will automatically resume at #{limit_expires_at.strftime '%H:%M'}"
```